v2/datastream-to-spanner/terraform/samples/mysql-end-to-end/README.md

```diff
@@ -460,7 +460,58 @@ mysql_database = {
 
 By default, the Dataflow job performs a like-like mapping between
 source and Spanner. Any schema changes between source and Spanner can be
-specified using the `session file`. To specify a session file -
+specified in multiple ways -
```

Comment:
nit: in any one of the following ways. or mention they are mutually exclusive in some way. IIRC it will fail if all of these are passed.